### PR TITLE
Add commit-fee-rate option for specifying individual fee rates

### PR DESCRIPTION
**Pull request by andrewtoth**
Closes https://github.com/casey/ord/issues/1487.

### CODE REVIEW

File: src/subcommand/wallet/inscribe.rs [hunk 1]
```diff
@@ -45,6 +50,14 @@ pub(crate) struct Inscribe {
 
 impl Inscribe {
   pub(crate) fn run(self, options: Options) -> Result {
+    if let Some(commit_fee_rate) = self.commit_fee_rate {
```
Comment (casey) [inline src/subcommand/wallet/inscribe.rs:53]: I think this check should be removed. There are actually reasons my one might want to use a low fee rate for the commit transaction. You could use a low fee on commit and a high fee on reveal, to encourage miners to include both or neither, using CPFP.

**Review by casey — COMMENTED** [review]
Wasn't able to do a full review yet, sorry for the delay! But wanted to note that we should probably allow commit-fee-rate to be less than fee-rate. See comment!

File: src/subcommand/wallet/inscribe.rs (hunk 1 above)
Comment (andrewtoth) [reply src/subcommand/wallet/inscribe.rs]: Removed

**Comment by casey** [comment]
This looks great! I pushed f04eb06 to make the help string shorter, hopefully it looks okay!

**Pull request merged**